### PR TITLE
Cam rebase

### PR DESCRIPTION
**Pull request by JeevakaPrabu**
@kalyankondapally @saranyagopal1

### CODE REVIEW

**Comment by JeevakaPrabu** [comment]
@saranyagopal1 It adds the needed formats but the usage flags not enough. Existing code can be modified or the patches done before can be brought forward as is.

File: cros_gralloc/gralloc0/gralloc0.cc [hunk 1]
```diff
@@ -101,8 +101,8 @@ static int gralloc0_alloc(alloc_device_t *dev, int w, int h, int format, int usa
 
 	if (!supported) {
 		cros_gralloc_error("Unsupported combination -- HAL format: %u, HAL flags: %u, "
-				   "drv_format: %4.4s, drv_flags: %llu",
-				   format, usage, reinterpret_cast<char *>(&descriptor.drm_format),
+				   "drv_format: %u, drv_flags: %llu",
```
Comment (tpalli) [inline cros_gralloc/gralloc0/gralloc0.cc]: I don't think this is a good change. Instead of descriptive text (like "AR24") we get a random number.

Comment (tpalli) [inline cros_gralloc/gralloc0/gralloc0.cc]: so my suggestion is to reorder the arguments but keep the char * cast in place like it was for %4.4s

**Comment by kalyankondapally** [comment]
@tpalli looks good ?

**Comment by tpalli** [comment]
yeah that is fine (although original char* cast would've worked as well :))

**Comment by kalyankondapally** [comment]
merged

**Pull request closed without merging**